### DISCUSSION TITLE
call the lychee API to upload images to my newly created album

### DISCUSSION THREAD

**Discussion by iihorse** [LycheeOrg/Lychee#2303] [Q&A] [answered]
Excuse me: How can I use the custom command of the picgo application to call the lychee API to upload images to my newly created album instead of the default unclassified album?

**Comment by ildyria** [comment, marked as answer]
I assume you are using https://github.com/ttionya/picgo-plugin-lychee/

I think their readme documentation has a pretty good to answer your question.

**Reply by iihorse** [reply]
ok，their readme documentation Solved my problem, thank you。
